debian/rules

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/make -f
 
+export DEB_BUILD_MAINT_OPTIONS=hardening=+all
+
 %:
```

Comment:
pie is not really passed into cmake properly, it is applying to shared libraries too.. But that seems to be OK. or at least if it isn't Debian will have to fix debhelper or the compiler...